internal/server/openvpn.go

```diff
@@ -141,3 +143,27 @@ func (h *openvpnHandler) getPortForwarded(w http.ResponseWriter) {
 		w.WriteHeader(http.StatusInternalServerError)
 	}
 }
+
+func (h *openvpnHandler) setPortForwarded(w http.ResponseWriter, r *http.Request) {
+	decoder := json.NewDecoder(r.Body)
+	encoder := json.NewEncoder(w)
```

Comment:
nit move this above line 164 where it's first used. I know other handler functions are ugly doing this as well, but since it's fresh new code 😸

Reply:
Your wish is my command :smile: (00dc345fda17d7ddcae4474c4c9bf1ce61f5af36)